runtime/tests/interpreter/runtimetype_test.go

```diff
@@ -505,24 +505,22 @@ func TestInterpretReferenceType(t *testing.T) {
       struct S {}
 	  entitlement X
 
-      let a = ReferenceType(authorized: true, type: Type<@R>())
-      let b = ReferenceType(authorized: false, type: Type<String>())
-      let c = ReferenceType(authorized: true, type: Type<S>())
+      let a = ReferenceType(entitlements: ["S.test.X"], type: Type<@R>())!
```

Comment:
Should the entitlements set maybe be `[Type]` instead of `[String]`?

Reply:
No, there is no runtime function to create a runtime entitlement type, and IMO there should not be. They aren't really types, and it would be confusing for you to be able to create a `Type` value for an entitlement (especially because they are "erased" at runtime).